src/utils/ws.ts

```diff
@@ -3,6 +3,8 @@ import { defineHandler } from "../handler.ts";
 import type { Hooks as WSHooks } from "crossws";
 import type { EventHandler } from "../types/handler.ts";
 
+export type { Hooks, Peer, AdapterInternal, Message } from "crossws";
```

Comment:
We need to alias-prefix this types with `WebSocket` from h3

Reply:
Prefixed with `WS` now to match the other `ws.ts` usage, but happy to adjust :)